themes/scipy_lectures/static/nature.css_t

```diff
@@ -601,11 +601,12 @@ div.body img {
 }
 
 /* Center figures that are larger than 100% of the figure */
-div.body div.figure {
+/* div.body div.figure {
     margin-left: -200px;
     margin-right: -200px;
     text-align: center;
-}
+  }
+*/
```

Comment:
I can tell. If you open a gallery webpage. for example:
`intro/matplotlib/auto_examples/index.html`
you will see that all thumbnails are on top of each other because the margin option pull each element by 200px back